sources/115/document.adoc

```diff
@@ -21,20 +21,25 @@ punctuation usage across multiple languages and writing systems.
 Metanorma is a semantic authoring system that supports multiple languages and
 the publication of multi-lingual output.
 
-Support of semantic representation using native punctuation in their respective
-languages is essential for the accurate portrayal of information and to meet
-cultural and linguistic expectations.
+A major component of Metanorma architecture is dedicated to providing presentation rendering
+for the semantic content it encodes. This means that Metanorma often composes punctuated
+text to  represent semantic text, and that punctuation needs to follow the norms of a target
+language. Metanorma requires a semantic framework to capture the various types of punctuation
+it will add to the text it composes, and to provide well-governed rendering of that punctuation
+in the languages it supports.
 
 This document defines a framework for semantic punctuation usage across multiple
 languages and writing systems.
 
 The document covers the following languages and writing systems:
 
-* Traditional Chinese (Taiwan/Hong Kong conventions)
-* Simplified Chinese (Mainland China conventions)
-* Japanese (including vertical text layout)
-* Korean (Hangul-specific rules)
-* (informative) Latin-script languages (English, Spanish, French, Russian, etc.)
+* CJK (East Asian ideographic writing systems)
+  * Traditional Chinese (Taiwan/Hong Kong conventions)
+  * Simplified Chinese (Mainland China conventions)
+  * Japanese (including vertical text layout)
+  * Korean (Hangul-specific rules)
```

Comment:
AI often generates invalid AsciiDoc, you’ll need to clean it up manually. AI only understands MarkDown.